proposals/numeric-intptr.md

```diff
@@ -0,0 +1,344 @@
+- From `int` to **`nint`**, `long`, `float`, `double`, or `decimal`.
+- From `uint` to **`nuint`**, `long`, `ulong`, `float`, `double`, or `decimal`.
+- **From `nint` to `long`, `float`, `double`, or `decimal`.**
+- **From `nuint` to `ulong`, `float`, `double`, or `decimal`.**
+- From `long` to `float`, `double`, or `decimal`.
+- From `ulong` to `float`, `double`, or `decimal`.
+- From `char` to `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `float`, `double`, or `decimal`.
+- From `float` to `double`.
+
+\[...]
+
+### 10.2.11 Implicit constant expression conversions
+
+An implicit constant expression conversion permits the following conversions:
+
+- A *constant_expression* of type `int` can be converted to type `sbyte`, `byte`, `short`, `ushort`, `uint`, **`nint`, `nuint`**, or `ulong`, provided the value of the *constant_expression* is within the range of the destination type.
+\[...]
+
+### 10.3.2 Explicit numeric conversions
+
+The explicit numeric conversions are the conversions from a *numeric_type* to another *numeric_type* for which an implicit numeric conversion does not already exist:
+
+- From `sbyte` to `byte`, `ushort`, `uint`, **`nuint`**, `ulong`, or `char`.
+- From `byte` to `sbyte` or `char`.
+- From `short` to `sbyte`, `byte`, `ushort`, `uint`, **`nuint`**, `ulong`, or `char`.
+- From `ushort` to `sbyte`, `byte`, `short`, or `char`.
+- From `int` to `sbyte`, `byte`, `short`, `ushort`, `uint`, **`nuint`**, `ulong`, or `char`.
+- From `uint` to `sbyte`, `byte`, `short`, `ushort`, `int`, **`nint`**, or `char`.
+- From `long` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `ulong`, or `char`.
+- **From `nint` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `nuint`, `ulong`, or `char`.**
+- **From `nuint` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `nint`, `long`, or `char`.**
+- From `ulong` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, or `char`.
+- From `char` to `sbyte`, `byte`, or `short`.
+- From `float` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, or `decimal`.
+- From `double` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, or `decimal`.
+- From `decimal` to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, or `double`.
+
+\[...]
+
+### 10.3.3 Explicit enumeration conversions
+
+The explicit enumeration conversions are:
+
+- From `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, `double`, or `decimal` to any *enum_type*.
+- From any *enum_type* to `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, `char`, `float`, `double`, or `decimal`.
+- From any *enum_type* to any other *enum_type*.
+
+# 11 Expressions
+
+#### 11.6.4.6 Better conversion target
```

Comment:
>11.6.4.6 Better conversion target [](http://example.com/codeflow?start=5&length=33)

Is there any change in this section?

Reply:
There would be a change inside the ellipsis ("Specifically: ..."), but I didn't spell it out. I think the general rule is clear enough.